server/types/socket.ts

```diff
@@ -16,6 +16,11 @@ export interface JoinEvent {
 	name: string;
 }
 
+export interface ModeratorChangeEvent {
```

Comment:
Can we add this to the `WebSocketEvent` union type so that the switch-case in `socket.controller.ts` is properly typed

Reply:
Oof yeah. Sorry I just switched laptops and didn't realize that I forgot to turn on the Deno typecheck settings.